.gemini/create_issues.sh

```
@@ -0,0 +1,844 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+RESULTS_FILE="/Users/ab/code/candelahq/candela/.gemini/issue_results.txt"
+> "$RESULTS_FILE"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)